src/openwith.js

```diff
@@ -39,8 +39,7 @@ OME.setOpenWithUrlProvider("omero_iviewer", function(selected, url) {
 
     // append image list if more than one image was selected
     url += '?images=';
-    for (var i=0;i<selected_count-1;i++)
-        url += selected[i].id + ',';
+    selected.map((img) => url += img.id + ',');
```

Comment:
Minor point: It's a little confusing to see ```map``` when you're not doing anything with the list returned. You could use ```forEach``` if you simply want to iterate.
Even nicer is to do something like this (which doesn't add the trailing comma).
```url += selected.map((img) => img.id).join(",");```

However, we should probably avoid arrow functions in ```openwith.js``` since that is not transpiled to ES5, so it will break in IE etc.